inst/WORDLIST

```diff
@@ -1,16 +1,19 @@
 CDISC
+cloneable
 Forkers
+formatters
+funder
 Getter
+getter
 Hoffmann
+iteratively
 JoinKeys
 Pre
-Reproducibility
-SCDA
-UI
-cloneable
-formatters
-funder
-iteratively
 pre
 repo
```

Comment:
is this necessary?

Reply:
I've regenerated the `WORDLIST` on 1dc2db4d , some of the items were removed.

Thanks :bow: